docs/html/reference/build-system/pyproject-toml.md

```diff
@@ -130,6 +130,13 @@ dealing with [the same challenges as pip has for legacy builds](build-output).
 
 ## Fallback Behaviour
 
+```{warning}
+The following snippet merely describes the fallback behavior. For valid
+examples of `pyproject.toml` to use with setuptools, please refer to
+[the setuptools documentation](
+https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use).
```

Comment:
```suggestion
https://setuptools.pypa.io/en/stable/userguide/quickstart.html#basic-use).
```

`latest` follows main (i.e. includes unreleased changes)

Reply:
Thanks, applied and squashed!